src/db/arby-order.ts

```diff
@@ -0,0 +1,49 @@
+import {
+  DataTypes,
+  Model,
+  ModelAttributes,
+  ModelOptions,
+  Sequelize,
+} from 'sequelize';
+import BigNumber from 'bignumber.js';
+import { ModelCtor } from 'sequelize/types/lib/model';
+
+enum ArbyOrderSide {
+  BUY,
+  SELL,
+}
+
+type ArbyOrderCreationAttributes = {
+  cexOrderId: string;
+  openDexOrderId: string;
+  side: ArbyOrderSide;
+  quantity: BigNumber;
+  price: BigNumber;
+  time: number;
+  profit: BigNumber;
+};
+
+type ArbyOrderAttributes = ArbyOrderCreationAttributes;
+
+export interface ArbyOrderInstance
+  extends Model<ArbyOrderAttributes, ArbyOrderCreationAttributes>,
+    ArbyOrderAttributes {}
+
+export function ArbyOrder(sequelize: Sequelize): ModelCtor<ArbyOrderInstance> {
+  const attributes: ModelAttributes<ArbyOrderInstance> = {
+    cexOrderId: { type: DataTypes.STRING, primaryKey: true },
+    openDexOrderId: { type: DataTypes.STRING, allowNull: false },
+    side: { type: DataTypes.TINYINT, allowNull: false },
+    quantity: { type: DataTypes.BIGINT, allowNull: false },
+    price: { type: DataTypes.BIGINT, allowNull: false },
+    time: { type: DataTypes.BIGINT, allowNull: false },
```

Comment:
Time as in unix timestamp?

Reply:
Yes, that's what I thought

Reply:
What about https://sequelize.org/master/class/lib/data-types.js~DATE.html?